server/status/recorder.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 const (
-	storeTimeSeriesNameFmt = "cr.store.%d.%s"
+	storeTimeSeriesNameFmt = "cr.store.%s.%d"
```

Comment:
These strings are supposed to be human readable. I also don't think there is any reason for them to be absolutely ordered by the StoreID; I don't expect consecutive StoreIDs to have a significance that would benefit from adjacency here.

I'll add a comment explaining the structure.